htdocs/openwebrx.js

```diff
@@ -535,16 +543,23 @@ function format_frequency(format, freq_hz, pre_divide, decimals) {
 var canvas_drag = false;
 var canvas_drag_min_delta = 1;
 var canvas_mouse_down = false;
+var canvas_mouse2_down = 0;
 var canvas_drag_last_x;
 var canvas_drag_last_y;
 var canvas_drag_start_x;
 var canvas_drag_start_y;
 
 function canvas_mousedown(evt) {
-    canvas_mouse_down = true;
-    canvas_drag = false;
-    canvas_drag_last_x = canvas_drag_start_x = evt.pageX;
-    canvas_drag_last_y = canvas_drag_start_y = evt.pageY;
+    if (evt.button > 0)
+        if (canvas_mouse2_down == 0)
+            canvas_mouse2_down = evt.button;
+    else {
+        canvas_mouse_down = true;
+        canvas_drag = false;
+        canvas_drag_last_x = canvas_drag_start_x = evt.pageX;
+        canvas_drag_last_y = canvas_drag_start_y = evt.pageY;
+    }
+
```

Comment:
Very well, I will invert the behavior, so that tuning is only available with the scroll wheel pushed (or right button pressed). Will this satisfy your desire not to change the current behavior of the interface?

Reply:
Inverted in the last commit.